tests_integration/snowflake_connector.py

```diff
@@ -9,24 +9,30 @@
 from contextlib import contextmanager
 
 _ENV_PARAMETER_PREFIX = "SNOWFLAKE_CONNECTIONS_INTEGRATION"
+SCHEMA_ENV_PARAMETER = f"{_ENV_PARAMETER_PREFIX}_SCHEMA"
+DATABASE_ENV_PARAMETER = f"{_ENV_PARAMETER_PREFIX}_DATABASE"
 
 
 def add_uuid_to_name(name: str) -> str:
     return f"{name}_{uuid.uuid4().hex}"
 
 
+@contextmanager
+def set_env(env_name: str, value: str):
+    os.environ[env_name] = value
+    yield
+    del os.environ[env_name]
```

Comment:
Why not use `mock.patch.dict(os.environ)`?

Reply:
Fixed in https://github.com/snowflakedb/snowflake-cli/pull/971